tests/e2e/test_vdbench_workload.py

```diff
@@ -0,0 +1,279 @@
+        if files:
+            sf_data['spec']['workload']['args']['files'] = files
+            target_results = target_results + '-' + str(files)
+        if file_size:
+            sf_data['spec']['workload']['args']['file_size'] = file_size
+            target_results = target_results + '-' + str(file_size)
+        if runtime:
+            sf_data['spec']['workload']['args']['runtime'] = runtime
+            target_results = target_results + '-' + str(runtime)
+        if pause:
+            sf_data['spec']['workload']['args']['pause'] = pause
+            target_results = target_results + '-' + str(pause)
+        if len(blocksize) > 0:
+            sf_data['spec']['workload']['args']['bs'] = blocksize
+            target_results = target_results + '-' + '_'.join(blocksize)
+
+        """
+            Calculating the size of the volume that need to be test, it should
+            be at least twice in the size then the size of the files, and at
+            least 100Gi.
+            since the file_size is in Kb and the vol_size need to be in Gb,
+            more calculation is needed.
+        """
+        vol_size = int((files * (width ** depth)) * file_size * 1.3)
+        log.info('number of files to create : {}'.format(
+            int(files * (width ** depth)))
+        )
+        log.info(f'The size of all files is : {vol_size}MB')
+        vol_size = int(vol_size / 1024)
+        if vol_size < 100:
+            vol_size = 100
+        sf_data['spec']['workload']['args']['storagesize'] = f'{vol_size}Gi'
+
+        log.info(sf_data)
+
+        timeout = 8640000  # 3600 (1H) * 24 (1D) * 100 = one hundred days
+
+        sf_obj = OCS(**sf_data)
+        sf_obj.create()
+        # wait for benchmark pods to get created - takes a while
+        for bench_pod in TimeoutSampler(
+            300, 10, get_pod_name_by_pattern, 'vdbench-client', 'my-ripsaw'
+        ):
+            try:
+                if bench_pod[0] is not None:
+                    vdbench_client_pod = bench_pod[0]
+                    break
+            except IndexError:
+                log.info('Benchmark client pod not ready yet')
+
+        bench_pod = OCP(kind='pod', namespace='my-ripsaw')
+        log.info('Waiting for VDBench benchmark to Run')
+        assert bench_pod.wait_for_resource(
+            condition=constants.STATUS_RUNNING,
+            resource_name=vdbench_client_pod,
+            sleep=30,
+            timeout=600
+        )
+        start_time = time.time()
+        while True:
+            logs = bench_pod.exec_oc_cmd(
+                f'logs {vdbench_client_pod}',
+                out_yaml_format=False
+            )
+            if 'Test Run Finished' in logs:
+                log.info('VdBench Benchmark Completed Successfully')
+                break
+
+            if timeout < (time.time() - start_time):
+                raise TimeoutError('Timed out waiting for benchmark to complete')
+            time.sleep(30)
+
+        """
```

Comment:
Please use # instead of docstring to comment.

Reply:
fixed.